gradle/shared-with-buildSrc/mirrors.settings.gradle.kts

```diff
@@ -71,7 +69,7 @@ fun normalizeUrl(url: String): String {
 
 fun overridesPluginPortalUrl() = providers.systemProperty(PLUGIN_PORTAL_OVERRIDE_URL_PROPERTY).forUseAtConfigurationTime().orNull != null
 
-if (!overridesPluginPortalUrl() && !isEc2Agent() && !isMacAgent() && !ignoreMirrors()) {
+if (isCI() && !overridesPluginPortalUrl() && !ignoreMirrors()) {
     // https://github.com/gradle/gradle-private/issues/2725
     // https://github.com/gradle/gradle-private/issues/2951
     System.setProperty(PLUGIN_PORTAL_OVERRIDE_URL_PROPERTY, "https://repo.grdev.net/artifactory/gradle-plugin-portal-prod/")
```

Comment:
Can't we pass the URL via an environment variable?

Reply:
I just noticed we have already defined `%gradle.plugins.portal.url%` in Root project, so we can just remove it.